students/sangyeon/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class User(models.Model): 
+    name         = models.CharField(max_length=50)
+    email        = models.EmailField(max_length=50, unique=True)
```

Comment:
```suggestion
    email        = models.CharField(max_length=50, unique=True)
```

상연님~~ 직전에 피드백 드렸던 email의 필드 타입이 수정되지 않은 것 같습니다.
다시 수정해주시고 다시 한번 더 migration 해주세요!

Reply:
Datefield만 보고 email 필드를 못봤네요..수정하였습니다!